frontend/src/components/eMIB/ActionViewTask.jsx

```diff
@@ -35,13 +52,29 @@ class ActionViewTask extends Component {
         </div>
         <hr style={styles.hr} />
         <div aria-label={LOCALIZE.ariaLabel.taskOptions}>
-          <button className="btn btn-primary" style={styles.editButton}>
+          <button
+            className="btn btn-primary"
+            style={styles.editButton}
+            onClick={this.showTaskDialog}
+          >
             {LOCALIZE.emibTest.inboxPage.emailCommons.editButton}
           </button>
           <button className="btn btn-danger">
             {LOCALIZE.emibTest.inboxPage.emailCommons.deleteButton}
           </button>
         </div>
+        <div>
+          <EditActionDialog
+            emailId={this.props.emailId}
+            emailSubject={this.props.emailSubject}
```

Comment:
Was going to ask if we need email subject for a task, but then saw that it is getting used further down, so good with this.

Reply:
Do you want me to add a comment for this one?